dmrunner/process.py

```diff
@@ -94,10 +92,10 @@ def services_healthcheck(shutdown_event, check_once=False):
 
                 # Connect to Postgres with default parameters - assume a successful connection means postgres is up.
                 try:
-                    psycopg2.connect(dbname='digitalmarketplace', user=getpass.getuser(), host='localhost').close()
+                    subprocess.run(['pg_isready', '-d', 'digitalmarketplace', '-U', getpass.getuser(), '-h', 'localhost'], check=True)
```

Comment:
This spits output directly to the console. We should probably use `subprocess.call` with stdout and stderr piped to null, like elsewhere during setup.

Reply:
![screen shot 2018-07-06 at 09 47 03](https://user-images.githubusercontent.com/2920760/42369338-8d4de9e2-8101-11e8-9950-b5dfa79eb2bc.png)